example-react-app/src/app/AppMenu.tsx

```diff
@@ -18,277 +18,283 @@ export const AppMenu = (props: AppMenuProps) => {
         screenId={"ExampleCustomScreen"}
```

Comment:
Please change commit prefix to `chore` and commit title to something like `add ComponentName`.